connectors/cli.py

```diff
@@ -89,14 +89,14 @@ async def _start_service(config, loop):
     services = [SyncService(config), JobCleanUpService(config)]
     for sig in (signal.SIGINT, signal.SIGTERM):
 
-        async def _shutdown(sig_name):
+        def _shutdown(sig_name):
             logger.info(f"Caught {sig_name}. Graceful shutdown.")
             for service in services:
                 logger.info(f"Shutdown {service.__class__.__name__}...")
-                await service.stop()
+                service.stop()
```

Comment:
maybe you want to try catch this, so if it fails we still `stop` others

Reply:
Good one, will do!